solr/core/src/java/org/apache/solr/response/transform/DocTransformers.java

```diff
@@ -88,4 +89,11 @@ public boolean needsSolrIndexSearcher() {
     return false;
   }
 
+  @Override
+  public void finish() {
+    for( DocTransformer a : children ) {
+      a.finish();
```

Comment:
I don't think it a way to design SPI (not A). Such silence should be either enforced with final/abstract constraints; or enforced in runtime. Imagine if some good transformer needs finish() to release a resource, but there is some badass sibling one. and it's enbaled from time-to-time at prod. requests. The instance will steadily leak, and it would be hard to find out why.